objects/authentication_factor.json

```diff
@@ -0,0 +1,82 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
+  "name": "authentication factor",
+  "extends": "object",
+  "attributes": {
+    "factor_device": {
+      "description": "Device used to complete the authentication request.",
+      "requirement": "recommended"
+    },
+    "factor_type": {
+      "enum": {
+        "1": {
+          "caption": "SMS"
+        },
+        "2 question": {
+          "caption": "Security Question"
+        },
+        "3": {
+          "caption": "Call"
```

Comment:
Should this be "Phone Call"?  "Call" seems ambiguous.

Reply:
Updated